price_service/server/src/listen.ts

```diff
@@ -440,6 +440,30 @@ export class Listener implements PriceStore {
       return false;
     }
 
+    // Check whether we have stale feeds and report false if we have too many.
```

Comment:
I think we can reword this comment to say something like "if too many price feeds are stale it probably means that the price service is not receiving messages from Wormhole at all and is essentially dead".

Reply:
👍